{{cookiecutter.github_project_name}}/tsconfig.json

```diff
@@ -7,10 +7,14 @@
     "moduleResolution": "node",
     "noEmitOnError": true,
     "noUnusedLocals": true,
-    "outDir": "../lib",
+    "outDir": "lib",
+    "resolveJsonModule": true,
+    "rootDir": "src",
     "skipLibCheck": true,
     "sourceMap": true,
     "strict": true,
-    "target": "es2015"
-  }
+    "target": "es2015",
+    "types": ["node"]
```

Comment:
I needed it because we do a `require()` in the code at https://github.com/jupyter-widgets/widget-ts-cookiecutter/blob/51e9fed8687e3b9cf1ed2fd307b7675e864f89ae/%7B%7Bcookiecutter.github_project_name%7D%7D/src/version.ts#L4. Perhaps we don't need that line?